packages/amplify-e2e-core/src/utils/nexpect.ts

```diff
@@ -307,7 +307,7 @@ function chain(context: Context): ExecutionContext {
         },
         name: '_send',
         shift: true,
-        description: "'[send] Y <CR>",
+        description: "'[send] N <CR>",
```

Comment:
nit: looks like this is just sending "N", not "N <CR>"

Reply:
I'll double check this. What struck me here was that the API says `sendNo` and it had `Y`.